profilegen/src/res/templates/spec_template.template

```diff
@@ -0,0 +1,30 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+
+require_relative 'lib/client'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'azure_mgmt_profile_<%= profile_version %>'
+  spec.version       = '<%= profile_ruby_version %>'
+  spec.authors       = 'Microsoft Corporation'
+  spec.email         = 'azrubyteam@microsoft.com'
+  spec.description   = 'Microsoft Azure Management Profile'
+  spec.summary       = 'Official Ruby client library to consume Microsoft Azure Management profile.'
+  spec.homepage      = 'https://aka.ms/azure-sdk-for-ruby'
+  spec.license       = 'MIT'
+
+  spec.files         = Dir['LICENSE.txt', 'lib/**/*']
+  spec.require_paths = ['lib']
+
```

Comment:
Do you need `spec.bindir` & `spec.executables` as we have it in all SDK ?

Reply:
We do not have any executables associated with the profile gems. So, it is not required. I do not have any idea why it was included in the SDK.